tests/installation/installer_desktopselection.pm

```diff
@@ -15,7 +15,7 @@ use testapi;
 sub run() {
     my %desktopkeys = (kde => "k", gnome => "g", xfce => "x", lxde => "l", minimalx => "m", textmode => "i");
     assert_screen "desktop-selection", 30;
-    my $d   = get_var("DESKTOP");
+    my $d = check_var('DESKTOP', 'awesome') ? "minimalx" : get_var('DESKTOP');
```

Comment:
or we can add support for something like `check_var('VAR', ['value1', 'value2']);`. Similar to `check_var_array` but the other way around.